backend/spec/requests/questions_spec.rb

```diff
@@ -1,157 +1,158 @@
-
-            it 'filters list of questions with difficulty level: medium' do
-                expect(JSON.parse(response.body)).not_to be_empty
-                expect(JSON.parse(response.body).size).to eq(3)
-            end
-        end
-
-        context 'when difficulty level: hard' do
-            before { get '/questions?difficulty_level=hard' }
-
-            it 'filters list of questions with difficulty level: hard' do
-                expect(JSON.parse(response.body)).not_to be_empty
-                expect(JSON.parse(response.body).size).to eq(2)
-            end
-        end
+  describe 'GET /questions?difficulty_level=:difficulty_level' do
+    let(:easy) { { difficulty_level: 'easy' } }
+    let(:medium) { { difficulty_level: 'medium' } }
+    let(:hard) { { difficulty_level: 'hard' } }
+
+    context 'when difficulty level: easy' do
+      before { get '/questions?difficulty_level=easy' }
+
+      it 'filters list of questions with difficulty level: easy' do
+        expect(JSON.parse(response.body)).not_to be_empty
+        expect(JSON.parse(response.body).size).to eq(5)
+      end
     end
 
-    describe 'POST /questions/:id' do
+    context 'when difficulty level: medium' do
+      before { get '/questions?difficulty_level=medium' }
 
-        context 'when user anwer is correct' do
-            before { post '/questions/1', params: { user_answer: 'Mammal' } }
+      it 'filters list of questions with difficulty level: medium' do
+        expect(JSON.parse(response.body)).not_to be_empty
+        expect(JSON.parse(response.body).size).to eq(3)
+      end
+    end
 
-            it 'returns a correct answer message' do
-                expect(response.body)
-                    .to match(/Yay! You're correct!/)
-            end
-        end
+    context 'when difficulty level: hard' do
+      before { get '/questions?difficulty_level=hard' }
+
+      it 'filters list of questions with difficulty level: hard' do
+        expect(JSON.parse(response.body)).not_to be_empty
+        expect(JSON.parse(response.body).size).to eq(2)
+      end
+    end
+  end
+
+  describe 'POST /questions/:id' do
+
+    context 'when user anwer is correct' do
+      before { post '/questions/1', params: { user_answer: 'Mammal' } }
+
+      it 'returns a correct answer message' do
+        expect(response.body)
+            .to match(/Yay! You're correct!/)
+      end
+    end
 
-        context 'when user answer is incorrect' do
-            before { post '/questions/1', params: { user_answer: 'Reptile' } }
+    context 'when user answer is incorrect' do
+      before { post '/questions/1', params: { user_answer: 'Reptile' } }
 
-            it 'returns a wrong answer message' do
-                expect(response.body)
-                    .to match(/Sorry, your answer is wrong. \:\(/)
-            end
-        end
+      it 'returns a wrong answer message' do
+        expect(response.body)
+            .to match(/Sorry, your answer is wrong. \:\(/)
```

Comment:
Very minor. This looks a bit odd, can you turn this into a one-liner test expectation?

e.g., `expect().to match()`